zap/src/test/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRFUnitTest.java

```diff
@@ -240,6 +240,49 @@ void shouldGetTokensFromInputFieldsIfIdAndNameIsAKnownTokenWithDifferentCase() {
             assertAntiCsrfToken(
                     tokens.get(1), message, KNOWN_TOKEN_2.toLowerCase(Locale.ROOT), "value2", 0);
         }
+
+        @Test
+        void
+                shouldGetTokensFromInputFieldsIfIdAndNameIncludesAKnownTokenAndPartialMatchingEnabled() {
+            given(antiCsrfParam.isPartialMatchingEnabled()).willReturn(true);
```

Comment:
Could you move this to after the comment? (Same comment for the following test.)